internal/engine/fswatch/gitmanager_test.go

```diff
@@ -0,0 +1,87 @@
+package fswatch
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/tilt-dev/tilt/internal/testutils"
+	"github.com/tilt-dev/tilt/internal/watch"
+
+	"github.com/tilt-dev/tilt/internal/k8s/testyaml"
+	"github.com/tilt-dev/tilt/internal/store"
+	"github.com/tilt-dev/tilt/internal/testutils/manifestbuilder"
+	"github.com/tilt-dev/tilt/internal/testutils/tempdir"
+	"github.com/tilt-dev/tilt/pkg/model"
+)
+
+func TestGitManager(t *testing.T) {
```

Comment:
no huge deal but maybe worth testing that an error reading the .git/HEAD file (see ln 126 above) doesn't break anything

Reply:
it doesn't seem worthwhile to mock out the file system just to test this

Reply:
Fair enough